### ISSUE TITLE
[UI] 

### ISSUE THREAD

**Issue by an3ast** [olive-editor/olive#1790]
**Commit Hash** 

**Platform** 

**Summary**

**Additional Information / Output**

Hi.
I have been working with a project for five hours straight and have been saving my project about every 15 minutes. An finally when i press the x to shut the program down i get a pop up message wich i dont exactly recall, but went along the lines that its been a long time scince youve saved your program, and i automatically assumed it wanted me to save the file befor closing so i press yes. What happend? Everything lost and im back at square one, five hours ago. I have no clut wheather it is my error or a bug or an issu with poor interface. Please if you are able to provide me with some magical way to accsess old logged saves please do.

**Comment by ThomasWilshaw** [comment]
Hmm, interesting. On Windows at least you can probably go to `C:\Users\UserName\AppData\Local\olivevideoeditor.org\Olive\autorecovery`. Not sure what it is in Linux

**Comment by an3ast** [comment]
Hi. Wow. Thanks for the quick reply. 
Is this possible to do on a macbok pro 2012?

**Comment by ThomasWilshaw** [comment]
I'm not a Mac user but it should be one of these locations I think:
`"~/Library/Application Support/<APPNAME>", "/Library/Application Support/<APPNAME>". "<APPDIR>/../Resources"`

@itsmattkc ?

**Comment by an3ast** [comment]
Didnt find anything initially and I’m am not very tech savvy/codefamiliar either. Anyway ill be sure to check it out in the morning. Best regards/ Andreas

**Comment by itsmattkc** [comment]
Olive 0.2 lets you browse and load auto-recoveries within the program, and keeps a substantial backlog of them so that true data loss is fairly rare.

But I have a sneaking suspicion you're using 0.1 which is no longer maintained and wasn't nearly as sophisticated. Yes, an autorecovery may be stored at `~/Library/Application Support/org.olivevideoeditor/Olive` but if not, I'm not sure there's much that can be done.

However, most recent versions of macOS store versions of files and let you revert to previous versions. If that's available, that might help you, but I'm not very familiar with how it works. You'll probably have to look it up.

**Comment by an3ast** [comment]
I have the 0.1 version unfortunately. But I have managed to find the suggested directory where the auto recovery files seems to be located. I have no clue how to use the files though. I have tried to drag the file into olive, but only the project box accepts it and showing only the filename och the dragged over file is visible and a red x to the left of it.

Any tips? //Andreas

**Comment by an3ast** [comment]
I solved the issue! :) I found out that the correct “Library”-folder was situated in the user-folder and hidden. I found that the Mac uses a special command (shift + command + .) to reveal hidden files. Once in the <Username>-folder I could easily locate the "application support”-folder and then access the “olivevideoeditor.org”-folder. After that I could locate the autosavefiles and just add .ove to the filename and press enter and then confirm the change of the filename. After this it was possible to open it as a functioning ove-project.

So I guess the directory would look something like this, for the not quite so tech savvy:
~/Users/<USERNAME>/Library/Application Support/olivevideoeditor.org”

Thanks you for the quick and professional support. Thanks to you I have five hours of my life back. 
Thank you again for your support. :) :) :) :) :)
